meta-luneos/recipes-multimedia/libcamera/libcamera_0.1.0.bb

```diff
@@ -0,0 +1,71 @@
+SUMMARY = "Linux libcamera framework"
```

Comment:
Add a comment that this is backorted from:
https://git.openembedded.org/meta-openembedded/commit/?id=4e0281797277cf44287de93618c51e4a219fac75

I assume it was copied as-is and shouldn't be merged to meta-webos-ports/master.

Reply:
Correct, this is only meant for kirkstone. on master, I expect to use the latest from meta-oe, which is where I took this content from anyway.

Reply:
kirkstone, langdale, mickledore :)

Reply:
```
--- ../meta-openembedded/meta-multimedia/recipes-multimedia/libcamera/libcamera_0.1.0.bb        2023-07-26 11:06:05.441889728 +0200
+++ meta-luneos/recipes-multimedia/libcamera/libcamera_0.1.0.bb 2023-07-27 11:52:05.283449804 +0200
@@ -23,13 +23,12 @@
 
 PACKAGES =+ "${PN}-gst"
 
-PACKAGECONFIG ??= ""
+PACKAGECONFIG ??= "gst"
 PACKAGECONFIG[gst] = "-Dgstreamer=enabled,-Dgstreamer=disabled,gstreamer1.0 gstreamer1.0-plugins-base"
 
-LIBCAMERA_PIPELINES ??= "auto"
-
 EXTRA_OEMESON = " \
-    -Dpipelines=${LIBCAMERA_PIPELINES} \
+    -Dpipelines=uvcvideo,simple,vimc \
+    -Dipas=vimc \
     -Dv4l2=true \
     -Dcam=enabled \
     -Dlc-compliance=disabled \
```

and not identical, it would be better to split this into backported recipe as-is and luneos specific modifications in .bbappend

Reply:
I'll update this.